release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1217,10 +1217,22 @@ OpenShift SDN network plugin was deprecated in 4.15 and 4.16. With this release,
 [id="ocp-4-17-cloud-cred-operator-bug-fixes_{context}"]
 ==== Cloud Credential Operator
 
+* Previously, secrets in the cluster were fetched in a single call.
```

Comment:
Looks like there are a couple of line breaks that shouldn't be there. 
"was" not "were" because the subject is "number of" 
```suggestion
* Previously, secrets in the cluster were fetched in a single call. When there was a large number of secrets, the API timed out. With this release, the Cloud Credential Operator fetches secrets in batches limited to 100 secrets. This change prevents timeouts when there is a large number of secrets in the cluster. (link:https://issues.redhat.com/browse/OCPBUGS-41233[*OCPBUGS-41233*])
```